templates/partials/modal.html.twig

```diff
@@ -20,27 +20,27 @@
              <ul>
                  <li>
                      <img src="{{ asset('images/icon-nodejs.svg') }}" /><br />
-                     <a href="https://the-example-app-nodejs.herokuapp.com/" rel="noopener" target="_blank">{{ 'hostedLabel'|trans }}</a> <br />
+                     <a href="https://the-example-app-nodejs.herokuapp.com/{{ state.usesCookieCredentials ? state.shareableLinkQuery : '' }}" rel="noopener" target="_blank">{{ 'hostedLabel'|trans }}</a> <br />
                      <a href="https://github.com/contentful/the-example-app.nodejs" rel="noopener" target="_blank">GitHub</a>
                  </li>
                  <li>
                      <img src="{{ asset('images/icon-dotnet.svg') }}" /><br />
-                     <a href="https://the-example-app-csharp.herokuapp.com/" rel="noopener" target="_blank">{{ 'hostedLabel'|trans }}</a> <br />
+                     <a href="https://the-example-app-csharp.herokuapp.com/{{ state.usesCookieCredentials ? state.shareableLinkQuery : '' }}" rel="noopener" target="_blank">{{ 'hostedLabel'|trans }}</a> <br />
                      <a href="https://github.com/contentful/the-example-app.csharp" rel="noopener" target="_blank">GitHub</a>
                  </li>
                  <li>
                      <img src="{{ asset('images/icon-ruby.svg') }}" /><br />
-                     <a href="https://the-example-app-rb.herokuapp.com/" rel="noopener" target="_blank">{{ 'hostedLabel'|trans }}</a> <br />
+                     <a href="https://the-example-app-rb.herokuapp.com/{{ state.usesCookieCredentials ? state.shareableLinkQuery : '' }}" rel="noopener" target="_blank">{{ 'hostedLabel'|trans }}</a> <br />
                      <a href="https://github.com/contentful/the-example-app.rb" rel="noopener" target="_blank">GitHub</a>
                  </li>
                  <li>
                      <img src="{{ asset('images/icon-php.svg') }}" /><br />
-                     <a href="https://the-example-app-php.herokuapp.com/" rel="noopener" target="_blank">{{ 'hostedLabel'|trans }}</a> <br />
+                     <a href="https://the-example-app-php.herokuapp.com/{{ state.usesCookieCredentials ? state.shareableLinkQuery : '' }}" rel="noopener" target="_blank">{{ 'hostedLabel'|trans }}</a> <br />
                      <a href="https://github.com/contentful/the-example-app.php" rel="noopener" target="_blank">GitHub</a>
                  </li>
                  <li>
                      <img src="{{ asset('images/icon-python.svg') }}" /><br />
-                     <a href="https://the-example-app-py.herokuapp.com/" rel="noopener" target="_blank">{{ 'hostedLabel'|trans }}</a> <br />
+                     <a href="https://the-example-app-py.herokuapp.com/{{ state.usesCookieCredentials ? state.shareableLinkQuery : '' }}" rel="noopener" target="_blank">{{ 'hostedLabel'|trans }}</a> <br />
```

Comment:
Just out of curriosity: is there ever a reason not to use the shareable link query? I was under the impression that the links are always the same ...

Reply:
If the app is using default setting, I don't think it's worth adding the default parameters to the URL. The result will be the same, but without the extra noise in the URL